### PR TITLE
Story/gat 1375

### PR DESCRIPTION
**Pull request by ThinkingMonkeyLtd**

### CODE REVIEW

File: src/pages/commonComponents/searchBar/SearchBar.js [hunk 1]
```diff
@@ -384,29 +384,8 @@ class SearchBar extends React.Component {
 
                             <Col lg={5} className='text-right'>
                                 <div className='nav-wrapper'>
```
Comment (kymmeh) [inline src/pages/commonComponents/searchBar/SearchBar.js:386]: Container, ClearButtonSvg and SearchInput are no-longer used.

File: src/pages/search/SearchPage.js [hunk 2]
```diff
@@ -1721,6 +1728,26 @@ class SearchPage extends React.Component {
         };
     }
 
+    handleSearch = e => {
+        e.preventDefault();
+        // const data = new FormData(e.target);
+        console.log(this.state.search);
```
Comment (kymmeh) [inline src/pages/search/SearchPage.js:1734]: log and comment ^^

File: src/pages/search/SearchPage.js [hunk 3]
```diff
@@ -1783,7 +1810,7 @@ class SearchPage extends React.Component {
                 {key === 'Datasets' && <DatasetSearchSort onSort={this.handleSort} sort={datasetSort} search={search} />}
                 {key === 'Datauses' && <DataUsesSearchSort onSort={this.handleSort} sort={dataUseRegisterSort} search={search} />}
                 {key === 'Collections' && <CollectionsSearchSort onSort={this.handleSort} sort={collectionSort} search={search} />}
-                {key === 'Papers' && <PapersSearchSort onSort={this.handleSort} sort={paperSort} search={search} />}
+                {key === 'Papers' && >apersSearchSort onSort={this.handleSort} sort={paperSort} search={search} />}
```
Comment (kymmeh) [inline src/pages/search/SearchPage.js:1813]: "P" has been deleted

File: src/pages/search/SearchPage.js [hunk 4]
```diff
@@ -1794,6 +1821,8 @@ class SearchPage extends React.Component {
         const filtersSelectionProps = this.getFiltersSelectionProps(preferenceFilters);
         const searchProps = this.getSearchProps(showSort, sortMenu, maxResults);
 
+        console.log('This ', this.state.search);
```
Comment (kymmeh) [inline src/pages/search/SearchPage.js:1824]: log ^^

File: src/pages/search/SearchPage.js [hunk 5]
```diff
@@ -1833,6 +1862,38 @@ class SearchPage extends React.Component {
...
+                                                                role='button'
+                                                            />
+                                                        )
+                                                    }
+                                                    value={this.state.searchFieldValue}
+                                                    onChange={this.handleSearchChange}
+                                                />
+                                            </Box>
+                                            <Box width='175px' mt={1}>
+                                                <Button type='submit'>Search</Button>
```
Comment (kymmeh) [inline src/pages/search/SearchPage.js:1890]: Button and search box look like they need a border radius of 0 on the sides where they touch.

**Review by kymmeh — CHANGES_REQUESTED** [review]

File: src/pages/search/SearchPage.js (hunk 3 above)
Comment (ThinkingMonkeyLtd) [reply src/pages/search/SearchPage.js]: PR was raised, conflicts / errors resolved in github but not tested locally. Changes are commited

File: src/pages/search/SearchPage.js (hunk 5 above)
Comment (ThinkingMonkeyLtd) [reply src/pages/search/SearchPage.js:1886]: We're going to finish this next sprint

**Review by kymmeh — APPROVED** [review]

**Review by kymmeh — APPROVED** [review]

**Review by kymmeh — APPROVED** [review]

**Review by cdjreekie — APPROVED** [review]

**Pull request merged**